Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts-buster-slim
```

Comment:
Let's go with `node:lts-alpine`, that's a little smaller in size:
https://hub.docker.com/layers/node/library/node/lts-alpine/images/sha256-c8f745d79647398912f9d9a1bc7161656c7cf60b3b6813bf9cb4ff682c9b784c?context=explore